solutions/testing/.github/workflows/playwright.yml

```diff
@@ -1,27 +1,25 @@
-name: Playwright Tests
+name: Playwright
+
 on:
-  push:
-    branches: [main, master]
-  pull_request:
-    branches: [main, master]
+  deployment_status:
```

Comment:
do you need `deployment_status:`?

Reply:
Yes, that's to make the action only run after there's a deployment, because the tests run on top of that deployment.